Assets/Scripts/UI/PickupPup/DogWorld/DogWorldSlot.cs

```diff
@@ -50,5 +50,15 @@ void handleScoutingDone()
     {
         this.Show();
     }
-        
+
+    /* BP Unsure if this is the preferred spot for this, but we apparently
+     * don't want any click noises for the dogs in the scene (yard/living room) */
+    void Start()
+    {
+        UISFXHandler sfxScript = GetComponent<UISFXHandler>();
+        sfxScript.clickEnabledSoundEvent = "";
```

Comment:
I see why you made them public, but generally we want to avoid exposing any class variables for direct access. I think a cleaner solution would be to write a method inside of `UISFXHandler`

`public void ToggleSoundsEnabled(bool areEnabled);` and then call that here